src/main/java/duke/DukeChatbot.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.command.Command;
+import duke.command.CommandParser;
+import duke.command.DukeInvalidCommandException;
+import duke.storage.StorageHandler;
+import duke.task.TaskHandler;
+import duke.ui.Ui;
+import javafx.application.Application;
```

Comment:
maybe can separate them into groups to make it clearer?

Reply:
I agree. I normally would separate these if it was my own project and I could set my own coding style. However, CheckStyle considers separating these as an error. I believe it is because separating them is not really a part of CS2103T's Java style guide. So, I will not separate them into groups even though I agree with you and I want to do that.